Sources/Orbit/Support/Components/SecureTextField.swift

```diff
@@ -9,6 +9,23 @@ struct SecureTextFieldStyle {
     let state: InputState
 }
 
+private class InsettedTextField: UITextField {
```

Comment:
`InsetTextField` - probably doesn't matter, but `set` is not a regular English verb... it's the same in all tenses.

Reply:
https://en.m.wiktionary.org/wiki/insetted
is it or do I get it wrong?

Reply:
There seems to be precious few references to that form... with quotes, I only get 169k hits on everything indexed by Google (and I still get "did you mean inserted?") as opposed to 104m for "inset". Granted, those would also contain the noun.

Meanwhile, these sites don't show "insetted":
- https://www.thefreedictionary.com/insetted (redirects to "inset")
- https://www.oxfordlearnersdictionaries.com/definition/english/inset_2

even https://en.wiktionary.org/wiki/inset shows it as "inset or insetted".

Even discounting all that though, playing with the prefix IMO shows that it should be "inset"... there is no "resetted" for example - just "reset". wiktionary of course includes "setted" for some reason, but doesn't even have a citation for that one. I'm beginning to suspect it as a reliable source now...

Reply:
I have the feeling that I've seen it before.
https://github.com/search?q=insetted&type=code
this is imo more related than google search.
but no issue with inset.

Reply:
Too bad we can't distinguish nouns here either, because just for "inset", I get 43 million results as opposed to less than 10k for "insetted".

> this is imo more related than google search.

Yeah, but should programmers be the authority on natural language? 😅 I'm willing to bet a lot of them are not native speakers - that's why I put the dictionary links in my reply.